### PR TITLE
Add area units acres and hectares. Add area density units.

### PR DESCRIPTION
**Pull request by cumminsjp**

### CODE REVIEW

File: UnitsNet.Tests/CustomCode/AreaDensityTests.cs [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+        }
+
+        protected override double KilogramsPerSquareMeterInOneKilogramPerSquareMeter
+        {
+            get { return 1; }
+        }
+
+        protected override double PoundsPerAcreInOneKilogramPerSquareMeter
+        {
+            get { return 8921.532921d; }
```
Comment (anjdreas) [inline UnitsNet.Tests/CustomCode/AreaDensityTests.cs]: Google begs to differ a bit with 8921.79, sounds a bit too much of difference?

https://www.google.no/search?q=Kilgrams+Per+Hectare+In+One+Kilogram+Per+Square+Meter&oq=Kilgrams+Per+Hectare+In+One+Kilogram+Per+Square+Meter&aqs=chrome..69i57.10816j0j7&sourceid=chrome&es_sm=93&ie=UTF-8#q=Pounds+Per+Acre+In+One+Kilogram+Per+SquareMeter

File: UnitsNet/Scripts/UnitDefinitions/AreaDensity.json [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+            "Localization": [
+                {
+                    "Culture": "en-US",
+                    "Abbreviations": [ "lb/ac" ]
+                }
+            ]
+        },
+        {
+            "SingularName": "KilogramPerHectare",
+            "PluralName": "KilgramsPerHectare",
```
Comment (anjdreas) [inline UnitsNet/Scripts/UnitDefinitions/AreaDensity.json]: Typo

File: UnitsNet.Tests/CustomCode/AreaDensityTests.cs [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+
+namespace UnitsNet.Tests.CustomCode
+{
+    public class AreaDensityTests : AreaDensityTestsBase
+    {
+        protected override double KilgramsPerHectareInOneKilogramPerSquareMeter
```
Comment (anjdreas) [inline UnitsNet.Tests/CustomCode/AreaDensityTests.cs]: Typo

**Comment by anjdreas** [comment]
The first merge commit should not be part of PR. Rebase your feature branch on top of origin/master instead.

Also, I would prefer if Area and AreaDensity units were two separate pull requests, but I won't require it.

**Comment by anjdreas** [comment]
Forgot to say thanks :-)

**Comment by cumminsjp** [comment]
yep,  i will do 2 pull req.

File: UnitsNet.Tests/CustomCode/AreaDensityTests.cs (hunk 1 above)
Comment (cumminsjp) [inline UnitsNet.Tests/CustomCode/AreaDensityTests.cs]: Unfortunately, I didn't save my spreadsheet where I was building my conversions but the 8921.53 value is definitely incorrect (for both us &  international acres). Also highlights additional changes needed for my area unit updates.

**Pull request closed without merging**